recipes/datacube-compute/recipe.yaml

```
@@ -0,0 +1,54 @@
+context:
+  name: datacube-compute
+  version: 0.0.6
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/auspatious/datacube-compute/archive/${{ version }}.tar.gz
+  sha256: 4706e086e4fb5049c88cfd28ea2dfda2f40c56146834e09fb59e8956d5be801a
+
+build:
+  number: 0
+  script:
+    env:
+      SETUPTOOLS_SCM_PRETEND_VERSION: ${{ version }}
+    content:
+      - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+      - ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  build:
+    - ${{ compiler('rust') }}
+    - cargo-bundle-licenses
+  host:
+    - pip
+    - python
+    - setuptools
+    - setuptools-rust
+    - setuptools-scm >=3.4
+    - wheel
+  run:
+    - numpy
+    - python
+    - xarray
+
+tests:
+  - python:
+      imports:
+        - datacube_compute
+      pip_check: true
+
+about:
+  summary: A library with fast implementations of algorithms for processing satellite images.
+  license: Apache-2.0
+  license_file:
+    - THIRDPARTY.yml
+    - LICENSE
+  homepage: https://github.com/auspatious/datacube-compute
+
+extra:
+  recipe-maintainers:
+    - weiji14
```

@alexgleith, ok if I put you down as a conda-forge recipe maintainer for `datacube-compute` (need a yes/no answer)? 

```suggestion
  recipe-maintainers:
    - alexgleith
    - weiji14
```

---

Yes!